code/__HELPERS/unsorted.dm

```diff
@@ -760,7 +760,10 @@ proc/anim(turf/location as turf,target as mob|obj,a_icon,a_icon_state as text,fl
 	var/image/progbar
 	for (var/i = 1 to numticks)
 		if(user.client)
-			progbar = make_progress_bar(i, numticks, target)
+			var/bar_target = target

```

Comment:
The default arguments are to catch any use of do_after that doesn't specify a target. Pretty much all of them should target _something_